cmd/neofs-node/config/contracts/config.go

```diff
@@ -0,0 +1,67 @@
+package contractsconfig
+
+import (
+	"fmt"
+
+	"github.com/nspcc-dev/neo-go/pkg/util"
+	"github.com/nspcc-dev/neofs-node/cmd/neofs-node/config"
+)
+
+const (
+	subsection = "contracts"
+)
+
+// Netmap returns value of "netmap" config parameter
+// from "contracts" section.
+//
+// Throws panic if value is not set.
+func Netmap(c *config.Config) util.Uint160 {
+	return contractAddress("netmap", c)
+}
+
+// Balance returns value of "balance" config parameter
+// from "contracts" section.
+//
+// Throws panic if value is not set.
+func Balance(c *config.Config) util.Uint160 {
+	return contractAddress("balance", c)
+}
+
+// Container returns value of "container" config parameter
+// from "contracts" section.
+//
+// Throws panic if value is not set.
+func Container(c *config.Config) util.Uint160 {
+	return contractAddress("container", c)
+}
+
+// Reputation returns value of "reputation" config parameter
+// from "contracts" section.
+//
+// Throws panic if value is not set.
+func Reputation(c *config.Config) util.Uint160 {
+	return contractAddress("reputation", c)
+}
+
+func contractAddress(name string, c *config.Config) util.Uint160 {
```

Comment:
i dont know why, by i intuitively want `Config` to be the first arg

Reply:
Yeah, It will look better, because first argument will be a broad one and second will be clarification.